data/giftcards.json

```diff
@@ -1,25 +1,25 @@
 [
     {
         "cardnumber" : "6036280000000000000",
-        "code" : "ANY",
```

Comment:
Change this to '100' as default value, see: 

https://docs.adyen.com/payment-methods/gift-cards/web-drop-in/#test-and-go-live


| Amount (last three digits) | Result Code | Refusal Reason           |
|---------------------------|-------------|--------------------------|
| 100                       | Authorised  |                          |
| 123                       | Refused     | Refused                  |
| 124                       | Refused     | Not enough balance       |
| 125                       | Refused     | Blocked Card             |
| 126                       | Refused     | Expired Card             |
| 130                       | Error       | Acquirer Error           |
| 134                       | Refused     | Invalid Pin              |
| 135                       | Refused     | Pin tries exceeded       |

Reply:
Good catch